libs/shared/flagd-core/src/lib/flagd-core.ts

```diff
@@ -135,7 +162,7 @@ export class FlagdCore implements Storage {
    * @throws {GeneralError} - If the variant specified in the flag is not found.
    */
   resolve<T extends FlagValue>(
-    type: FlagValueType,
+    type: FlagValueType | 'any',
```

Comment:
Would undefined also work here instead of any? - if we need `'any'` i would suggest to place it in a constant or something like this, to make it more accessible and understandable.

Reply:
This isn't an `any` type in the TypeScript sense. It's a string representation of the flag type. During bulk evaluation, we don't care what type it is because it will be enforced later by the SDK.

I'm happy to use a different name but I struggled to think of anything better.

Reply:
i am fine with `any` but it would be cool to use it as a type rather than a string like 
```
type AnyFlagValueType = 'any';

and then we use the this type, rather than the string. the string has something magical to it. where as a defined type is easier understood, and can be better documented

Reply:
Can we instead make an overload that omits this param?

Reply:
I've refactored this part to remove the `any` type. Thanks for your feedback.